resources/functionMap.php

```diff
@@ -5166,7 +5166,7 @@
 'ImagickPixel::clear' => ['bool'],
 'ImagickPixel::clone' => ['void'],
 'ImagickPixel::destroy' => ['bool'],
-'ImagickPixel::getColor' => ['array', 'normalized='=>'bool'],
+'ImagickPixel::getColor' => ['array', 'normalized='=>'int'],
```

Comment:
Allow only 0, 1 or 2 integers
```suggestion
'ImagickPixel::getColor' => ['array', 'normalized='=>'0|1|2'],
```